pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```diff
@@ -1,21 +1,18 @@
 { ffmpeg_6-full
-, nv-codec-headers-12
 , chromaprint
 , fetchFromGitHub
 , lib
 }:
 
-(ffmpeg_6-full.override {
-  nv-codec-headers-11 = nv-codec-headers-12;
-}).overrideAttrs (old: rec {
+ffmpeg_6-full.overrideAttrs (old: rec {
```

Comment:
@justinas while we're here: What are you using ffmpeg-full here? What features are missing in the regular ffmpeg configuration? Full is intended for power users to optionally install into their systems with no regard for closure size, not for other programs to use.